source/numerics/vector_tools_project.inst.in

```diff
@@ -67,3 +67,40 @@ for (VEC : SERIAL_VECTORS ; deal_II_dimension : DIMENSIONS; deal_II_space_dimens
     \}
 #endif
 }
+
+for (deal_II_dimension : DIMENSIONS)
```

Comment:
The instantiation lists are listed in `cmake/config/template-arguments.in`, which get processed by `cmake/scripts/expand_instantiations.cc`. The rules for that are contained in `cmake/macros/macro_expand_instantiations.cmake`. However, I have never tried to manually add an instantiation not included in `template-arguments.h.in` in a way similar to what you propose. We could in principle add a new list to `template-arguments.h.in` and use that for instantiating. @tamiko is there another way?